doc/api/stream.md

```diff
@@ -2321,6 +2438,38 @@ const myDuplex = new Duplex({
 });
 ```
 
+When using pipeline:
+
+```js
+const { Transform } = require('stream');
```

Comment:
Should this destructure `pipeline` from the module too, for use below? It seems a bit inconsistent to put some, but not all, of the imports at the top of the example